production/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.4'
+
+services:
+  nodepki:
+    image: adito/nodepki-docker:latest
+    restart: unless-stopped
+    env_file:
+      - .env
+    environment:
+      TZ: Europe/Berlin
+    healthcheck:
+      test: ["CMD-SHELL", "curl --fail http://localhost:$${CA_WEBCLIENT_SERVER_PORT} || exit 1"]
```

Comment:
Double dollar? Is that right?

Reply:
Well, both should work. A single $ or double $. Double $ are required if you do not want Docker Compose to replace the variable with the effective value of it **at start time** (escaping the variable). Docker Compose replaces a double $ at start time with a single $ which basically means we use the environment variable ${CA_WEBCLIENT_SERVER_PORT} inside our curl health check. If we would only configure „${CA_WEBCLIENT_SERVER_PORT}“ (single $) inside our health check, Docker Compose would replace this variable with the value (e.g. 5000) before the Docker service is started. The effective docker-compose.yml file can be viewed by using the „docker-compose config“ command.

Regards,
Philip